api/data_refinery_api/serializers.py

```diff
@@ -790,49 +791,50 @@ class Meta:
                         }
                     }
 
+class CompendiumResultOrganismSerializer(serializers.ModelSerializer):
 
-class CompendiaSerializer(serializers.ModelSerializer):
-    organism_name = serializers.CharField(source='compendia_organism.name', read_only=True)
+    def to_representation(self, value):
+        return value.name
 
     class Meta:
-        model = ComputedFile
+        model = Organism
+        fields = '__all__'
```

Comment:
do we need anything besides `name`?

Reply:
removed and used `serilizers.StringRelatedField`